src/containers/Storage/DiskStateProgressBar/DiskStateProgressBar.tsx

```diff
@@ -53,7 +57,7 @@ export function DiskStateProgressBar({
 
     const mods: Record<string, boolean | undefined> = {inverted};
 
-    const color = severity && severityToColor[severity];
+    const color = severity !== undefined && severityToColor[severity];
```

Comment:
Grey color corresponds to 0 severity, so check `severity &` is not valid here

Reply:
good catch, thanks